audit/format.go

```diff
@@ -284,11 +294,13 @@ func (f *AuditFormatter) FormatResponse(
 	respEntry := &AuditResponseEntry{
 		Type:  "response",
 		Error: errString,
-
 		Auth: AuditAuth{
-			DisplayName: auth.DisplayName,
-			Policies:    auth.Policies,
-			Metadata:    auth.Metadata,
+			ClientToken:   auth.ClientToken,
+			Accessor:      auth.Accessor,
+			DisplayName:   auth.DisplayName,
+			Policies:      auth.Policies,
+			Metadata:      auth.Metadata,
+			RemainingUses: req.ClientTokenRemainingUses,
```

Comment:
I think we had too. At the moment I don't remember the context around it; it may be in comments in the PR that introduced that change.